Source/URLEncodedFormEncoder.swift

```diff
@@ -60,6 +60,8 @@ public final class URLEncodedFormEncoder {
         case noBrackets
         /// Brackets containing the item index are appended. This matches the jQuery and Node.js behavior.
         case indexInBrackets
+        /// Provide a custom array key encoding with the given closure.
+        case custom((String, Int) -> String)
```

Comment:
```suggestion
        case custom((_ key: String, _ index: Int) -> String)
```

Internal labels make this a bit nicer to read.